apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/TableSchemaBuilder.cs

```diff
@@ -40,10 +41,27 @@ public sealed class TableSchemaBuilder : IEnumerable
         public void Add(TableFieldSchema field)
         {
             GaxPreconditions.CheckNotNull(field, nameof(field));
-
             _fields.Add(field);
         }
 
+        /// <summary>
+        /// Modifies the field with <paramref name="fieldName"/> by running the given action.
+        /// </summary>
+        /// <param name="fieldName">The name of the field to be modified. Must not be null.</param>
+        /// <param name="fieldModifier">An action to be performed on the given field.</param>
+        /// <returns>
+        /// This <see cref="TableSchemaBuilder"/> after having applied the modification. For the purposes of method chaining.
+        /// Note: If <paramref name="fieldName"/> or <paramref name="fieldName"/> is null, it will throw an exception.
+        /// </returns>
+        public TableSchemaBuilder ModifyField(string fieldName, Action<TableFieldSchema> fieldModifier)
+        {
+            GaxPreconditions.CheckNotNull(fieldName, nameof(fieldName));
+            GaxPreconditions.CheckNotNull(fieldModifier, nameof(fieldModifier));
+            var field = _fields.FirstOrDefault(field => field.Name == fieldName) ?? throw new ArgumentException($"Field '{fieldName}' not found in the schema.");
```

Comment:
Might as well add a second argument to the ArgumentException constructor, of `nameof(fieldName)`. (Again, sorry for not suggesting this before.)

Reply:
np Jon updated it.